src/ibm/hypervisor-network-mgr-src/hyp_ethernet_interface.cpp

```diff
@@ -27,12 +27,12 @@ constexpr auto BIOS_OBJPATH = "/xyz/openbmc_project/bios_config/manager";
 constexpr auto BIOS_MGR_INTF = "xyz.openbmc_project.BIOSConfig.Manager";
 
 // The total number of vmi attributes defined in biosTableAttrs
-// currently is 17:
-// 4 attributes of interface 0 - ipv4 address
-// 4 attributes of interface 0 - ipv6 address
-// 4 attributes of interface 1 - ipv4 address
-// 4 attributes of interface 1 - ipv6 address
-constexpr auto BIOS_ATTRS_SIZE = 17;
+// currently is 17 in case of systems with 2 eth interfaces and 9 with single
+// eth interface 4 attributes of interface 0 - ipv4 address 4 attributes of
+// interface 0 - ipv6 address 4 attributes of interface 1 - ipv4 address 4
+// attributes of interface 1 - ipv6 address
+constexpr auto BIOS_ATTRS_SIZE_WITH_IF1 = 17;
```

Comment:
thinking if we can get rid of these size based checks and just check attributes presence directly as we have discussed already.
If we are not going to use this code for p11 then we can skip 
@sunharis what do you suggest?

Reply:
@raviteja-b 
I am trying with HashMap<attribut_name, default_value>  approach as well , which I discussed with you yesterday . 
but with if attribute names change in future it will be an issue .